src/panels/Scene/GlobeLayers/GlobeLayersGroup.tsx

```diff
@@ -0,0 +1,47 @@
+import { Group, Paper, Space } from '@mantine/core';
+
+import { CollapsableContent } from '@/components/CollapsableContent/CollapsableContent';
+import { Property } from '@/components/Property/Property';
+import { useAppSelector } from '@/redux/hooks';
+import { displayName } from '@/util/propertyTreeHelpers';
+
+import { LayerList } from './LayersList';
+
+interface Props {
+  uri: string;
+  globe: string;
+  icon?: React.ReactNode;
+}
+
+export function GlobeLayerGroup({ uri, globe, icon }: Props) {
+  const propertyOwner = useAppSelector(
+    (state) => state.propertyOwners.propertyOwners[uri]
+  );
+
+  if (!propertyOwner) {
+    throw Error(`No property owner found for uri: ${uri}`);
+  }
+
+  const layers = propertyOwner?.subowners || [];
+  const properties = propertyOwner?.properties || [];
+
+  return (
+    <CollapsableContent
+      title={
+        <Group gap={'xs'}>
+          {icon && icon}
+          {displayName(propertyOwner!)}
+        </Group>
+      }
+      noTransition
+    >
+      <Paper p={'xs'}>
+        <LayerList layers={layers} layerGroup={propertyOwner.identifier} globe={globe} />
+        <Space h={'xs'} />
+        {properties.map((p) => (
+          <Property key={p} uri={p} />
```

Comment:
This component already has a `uri` variable. We should avoid reusing the name in this case. 

But if it is unclear, it could be renamed to e.g. `pUri` or `propertyUri`. Personally I preferred having it as short as possible here, though, since the variable is so local

Reply:
It's not that big of a deal but since `properties` here is a list of strings and not the type `Property` I think that's where my confusion comes from. At first glance, it looks like we're mapping over property objects but we only store the URI:s. But yeah I guess we should decide on that because I know other places where we use the same prop name and argument